apps/web/src/data/ecosystem.json

```diff
@@ -4214,5 +4214,13 @@
     "description": "The friendly crypto wallet built for DeFi & NFTs.",
     "url": "https://phantom.app/",
     "imageUrl": "/images/partners/phantom.svg"
+  },
+  {
+    "name": "20lab",
+    "description": "20lab.app is the most customizable ERC-20, SPL & Sui token generator with over 17 extendable features. Test our generator for free and manage functions via token owner dashboard.",
```

Comment:
Can you please remove the superlative and change "the most customizable" to "a customizable"